internal/core/data/v2/controller/http/event.go

```diff
@@ -67,8 +70,9 @@ func (ec *EventController) AddEvent(w http.ResponseWriter, r *http.Request) {
 
 	// map Event models to AddEventResponse DTOs
 	var addResponses []interface{}
+	var successAddEventReqDTOs []requestDTO.AddEventRequest
 	for i, e := range events {
-		newId, err := application.AddEvent(e, ctx, ec.dic)
+		newId, err := application.AddEvent(e, profileName, deviceName, ctx, ec.dic)
```

Comment:
`newId` can be removed and just use `e.Id`

Reply:
updated.